sdk-setup/src/oomadvice

```diff
@@ -111,7 +111,7 @@ subprocess()
 
 set_defaults()
 {
-    OPT_OOM_ADJ=10
+    OPT_OOM_SCORE_ADJ=10
```

Comment:
oom_adj range was from -17 to 15, oom_score_adj range is from -1000 to 1000. Should we adjust the default accordingly?

Reply:
good catch! I thought they we just aligning the name. Adjusted proportionally.